src/server.jl

```diff
@@ -31,23 +17,22 @@ end
 
 
 """
-    file_changed_callback(filepath, ev)
+    file_changed_callback(filepath::String)
 
-Callback that gets fired once a change to a file `filepath` is detected (FileEvent `ev`).
+Function reacting to the change of files.
 """
-function file_changed_callback(filepath::AbstractString, ev::FileWatching.FileEvent)
-    # only do something if file was changed ONLY
-    if ev.changed && !ev.renamed && !ev.timedout
-        if lowercase(splitext(filepath)[2]) ∈ (".html", ".htm")
-            # if html file, update viewers of this file only
-            update_and_close_viewers!(WS_HTML_FILES[filepath])
-        else
-            # otherwise (e.g. modification to a CSS file), update all viewers
-            foreach(update_and_close_viewers!, values(WS_HTML_FILES))
-        end
+function file_changed_callback(filepath::String)
+    println("ℹ [LiveUpdater]: Reacting to change in file '$filepath'...")
```

Comment:
same

Reply:
+ you probably want to have an overarching `const   VERBOSE = Ref{Bool}(true)`   which is toggled by a function and then have `VERBOSE.x && println...`

Reply:
For verbosity cf. #16.